src/third_party/highlightjs/highlightjs/languages/java.js

```diff
@@ -0,0 +1,176 @@
+// https://docs.oracle.com/javase/specs/jls/se15/html/jls-3.html#jls-3.10
```

Comment:
Missing license header

Reply:
This is generated by the highlighter.js build, license is below...